notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<p>i still think this step is not needed because they will eventually do something like </p><p> </p><pre class="ql-syntax" spellcheck="false">preds = model.fit(test)
</pre><p> </p><p>and then can inject the preds into the submission dataframe. adding the preds as a column to the test dataframe feels a bit unusual to me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>